src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/WidgetCSV.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
were these files made in 2022 or 2023? some files below have 2023 - lets be consistent and pick one for this pr

Reply:
As the WidgetCSV tests are merged now, this PR holds only required files.